apps/remote_control/test/lexical/remote_control/build/document/compilers/eex_test.exs

```diff
@@ -26,6 +26,17 @@ defmodule Lexical.RemoteControl.Build.Document.Compilers.EExTest do
     Document.new("file:///file.eex", content, 0)
   end
 
+  setup_all do
+    prev_compiler_options = Code.compiler_options()
+    Code.compiler_options(parser_options: [columns: true, token_metadata: true])
```

Comment:
should we just call `Build.State.set_compiler_options()` ?

Reply:
yeah, that's better.